test/functional/Java20andUp/src/org/openj9/test/jep434/upcall/UpcallMethodHandles.java

```diff
@@ -1838,7 +1838,7 @@ public static double addDoubleAndIntDoubleFromStruct(double arg1, MemorySegment
 		/* The size of [int, double] on AIX/PPC 64-bit is 12 bytes without padding by default
 		 * while the same struct is 16 bytes with padding on other platforms.
 		 */
-		double structElem2 = (isAixOS) ? arg2.get(JAVA_DOUBLE.withBitAlignment(32), 4) : arg2.get(JAVA_DOUBLE, 8);
+		double structElem2 = (isAixOS) ? arg2.get(JAVA_DOUBLE, 4) : arg2.get(JAVA_DOUBLE, 8);
```

Comment:
The parentheses in `(isAixOS)` are not needed; this can now be simplified
```java
		double structElem2 = arg2.get(JAVA_DOUBLE, isAixOS ? 4 : 8);
```
Likewise for many similar declarations of `structElem2`.

Reply:
Agreed and fixed it everywhere with `structElem2` in the case of AIX.

Reply:
It seems you fixed the Java21andUp test code, but not this.

Reply:
Yes. Just fixed in Java20andUp.